File: packages/create-react-native-library/templates/java-view-mixed/android/src/oldarch/{%- project.name %}ViewManagerSpec.java
package com.<%- project.package %>;

import android.view.View;

import androidx.annotation.Nullable;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.SimpleViewManager;

public abstract class <%- project.name -%>ViewManagerSpec<T extends View> extends SimpleViewManager<T> {
  public abstract void setColor(T view, @Nullable String value);
}

File: packages/create-react-native-library/templates/kotlin-view-library-mixed/android/src/oldarch/{%- project.name %}ViewManagerSpec.kt
package com.<%- project.package %>

import android.view.View
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.uimanager.SimpleViewManager

abstract class <%- project.name -%>ViewManagerSpec<T : View> : SimpleViewManager<T>() {
  abstract fun setColor(view: T?, value: String?)
}
